docs/src/api.md

```diff
@@ -6,25 +6,25 @@ DocTestSetup  = quote
 end
 ```
 
-## Modules
+### Modules
```

Comment:
I just added common header for "### Modules", "### Constants": "## Components listing".

Because there is also second level "## Documentation" header with autodocs